### ISSUE TITLE
Errors when using generate label

### ISSUE THREAD

**Issue by sunwell1994** [shvdiwnkozbw/Multi-Source-Sound-Localization#8]
Dear author, I have several questions on generating labels

1. in ./mobile_crnn/generat_label.py, 
- should there be the ident for the for loop?
- should we use the Spec.h5 instead of Audio.h5 
- why is the aggregation process different generate_labelv.py?
2. for generate_labelv.py
- which resnet to use, the one in ./resnet or pretrained resnet from torch
- why set top k to be 0?

Thanks for your answer. It is not trivial to run the files directly. It will be best if the labels are provided. Thanks.

**Comment by zjsong** [comment]
Hi @shvdiwnkozbw , thanks for sharing the code.

In the following command, do we need to insert keyword `largest=False` into `torch.topk`? Because the default setting is `largest=True`...

https://github.com/shvdiwnkozbw/Multi-Source-Sound-Localization/blob/8fa5856c42c6a468b51b404bb1883d0ade70bf06/generate_labelv.py#L16

**Comment by shvdiwnkozbw** [comment]
> Dear author, I have several questions on generating labels
> 
> 1. in ./mobile_crnn/generat_label.py,
> 
> * should there be the ident for the for loop?
> * should we use the Spec.h5 instead of Audio.h5
> * why is the aggregation process different generate_labelv.py?
> 
> 1. for generate_labelv.py
> 
> * which resnet to use, the one in ./resnet or pretrained resnet from torch
> * why set top k to be 0?
> 
> Thanks for your answer. It is not trivial to run the files directly. It will be best if the labels are provided. Thanks.

Thanks for pointing out, I made some mistakes when reimplementing this code for label generation.
Yes, there should be indent, and use spec.h5 for label generation.
When generate label_v, we use imagenet-pretrained resnet-50 in torchvision for label prediction. In order to align the audio and visual categories, we need to aggregate the category predictions from 1000 imagenet categories into 7 general classes in audioset.

**Comment by shvdiwnkozbw** [comment]
> Hi @shvdiwnkozbw , thanks for sharing the code.
> 
> In the following command, do we need to insert keyword `largest=False` into `torch.topk`? Because the default setting is `largest=True`...
> 
> https://github.com/shvdiwnkozbw/Multi-Source-Sound-Localization/blob/8fa5856c42c6a468b51b404bb1883d0ade70bf06/generate_labelv.py#L16

Yes, it is mistake in reimplementing this script, it should be largest=False. Thanks!
